pkg/frontend/tae_txn.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2021 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package frontend
+
+import (
+	"context"
+
+	"github.com/matrixorigin/matrixone/pkg/pb/txn"
+	"github.com/matrixorigin/matrixone/pkg/txn/rpc"
+	"github.com/matrixorigin/matrixone/pkg/vm/engine/tae/moengine"
+)
+
+type wrappedTAEEngine struct {
```

Comment:
Same as above

Reply:
Can you provide the mechanism that the frontend can get the transaction handler without careing the storage is TAE or DN? After DN is done in the future, the frontend does not see the tae again.  I do not need to revise the code involved with the transaction handler again.

Reply:
> Can you provide the mechanism that the frontend can get the transaction handler without careing the storage is TAE or DN? After DN is done in the future, the frontend does not see the tae again. I do not need to revise the code involved with the transaction handler again.

I'm working on this